assets/alertmanager/alertmanager.yaml

```diff
@@ -30,7 +30,9 @@ spec:
     - -http-address=
     - -email-domain=*
     - -upstream=http://localhost:9093
-    - '-openshift-sar={"resource": "namespaces", "verb": "get"}'
+    - '-openshift-sar=[{"resource": "namespaces", "verb": "get"}, {"resource": "alertmanagers",
```

Comment:
To avoid granting higher permissions than necessary, I would suggest that
1) we use a non-existing resource name 
2) the verb is "patch" instead of "*"

This way the user couldn't do any change to the cluster even if they wanted to.

```
{"resource": "alertmanagers", "resourceAPIGroup": "monitoring.coreos.com", "namespace": "openshift-monitoring", "name": "not-existing","verb": "patch"}
```